File: Python/Program/RiwayatTurnamen.py
# File: RiwayatTurnamen.py
class RiwayatTurnamen:
    def __init__(self, nama_turnamen="", tahun=0, peringkat_juara=0):
        self._nama_turnamen = nama_turnamen
        self._tahun = tahun
        self._peringkat_juara = peringkat_juara
    
    def get_nama_turnamen(self): return self._nama_turnamen
    def get_tahun(self): return self._tahun
    def get_peringkat_juara(self): return self._peringkat_juara

File: Java/Program/RiwayatTurnamen.java
public class RiwayatTurnamen {
    private String namaTurnamen;
    private int tahun;
    private int peringkatJuara;

    public RiwayatTurnamen(String nama, int tahun, int peringkat) {
        this.namaTurnamen = nama;
        this.tahun = tahun;
        this.peringkatJuara = peringkat;
    }

    public String getNamaTurnamen() { return this.namaTurnamen; }
    public int getTahun() { return this.tahun; }
    public int getPeringkatJuara() { return this.peringkatJuara; }
}